resources/schemas.json

```diff
@@ -734,6 +734,114 @@
                 },
                 "title": "AuthenticateResponseView",
                 "description": "the authentication response"
+            },
+            "CreateLocRequestView": {
+                "type": "object",
+                "properties": {
+                    "ownerAddress": {
+                        "type": "string",
+                        "description": "The SS58 address of the legal officer that will own the LOC upon acceptance"
+                    },
+                    "requesterAddress": {
+                        "type": "string",
+                        "description": "The SS58 address of the LOC requester"
+                    },
+                    "description": {
+                        "type": "string",
+                        "description": "A description of the LOC"
+                    }
+                },
+                "title": "CreateLocRequestView",
+                "description": "A LOC Request to create"
+            },
+            "LocRequestView": {
+                "type": "object",
+                "properties": {
+                    "ownerAddress": {
+                        "type": "string",
+                        "description": "The SS58 address of the legal officer that will own the LOC upon acceptance"
+                    },
+                    "requesterAddress": {
+                        "type": "string",
+                        "description": "The SS58 address of the LOC requester"
+                    },
+                    "description": {
+                        "type": "string",
+                        "description": "A description of the LOC"
+                    },
+                    "createdOn": {
+                        "type": "string",
+                        "format": "date-time",
+                        "description": "The creation timestamp"
+                    },
+                    "decisionOn": {
+                        "type": "string",
+                        "format": "date-time",
+                        "description": "The decision timestamp (if status is 'STARTED' or 'REJECTED')"
+                    },
+                    "id": {
+                        "type": "string",
+                        "format": "uuid",
+                        "description": "The ID of the LOC request"
+                    },
+                    "status": {
+                        "type": "string",
+                        "description": "The request's status",
+                        "enum": [
+                            "STARTED",
```

Comment:
I know that the US mentions "Started" but wouldn't "Accepted" be better (by symmetry with other requests)?

Reply:
I guess more statuses will come, for instance "CLOSED", etc. To be discussed.

Reply:
Statuses are renamed to "OPEN",  "REQUESTED", "REJECTED"